src/container/container_iv.c

```diff
@@ -383,6 +383,11 @@ cont_iv_ent_update(struct ds_iv_entry *entry, struct ds_iv_key *key,
 		/* If the uuid is NULL, then it will delete(invalidate) all
 		 * entries (containers under the pool) of the tree.
 		 */
+		if (entry->iv_class->iv_class_id == IV_CONT_CAPA) {
+			rc = ds_cont_tgt_close(civ_key->cont_uuid);
+			if (rc)
+				return rc;
+		}
```

Comment:
Should these line be added before the comment from line 383 to 385? The comment seems to describe line 391 and downward.

Reply:
This comment issue is here.